django_quickbooks/processors/base.py

```diff
@@ -6,34 +6,36 @@
 from django_quickbooks import QBXML_RESPONSE_STATUS_CODES
 from django_quickbooks.decorators import realm_connection
 from django_quickbooks.exceptions import QBXMLParseError, QBXMLStatusError
+from django_quickbooks import utils
 
 
 class ResponseProcessor:
     resource = None
     op_type = None
     obj_class = None
 
-    def __init__(self, response, hresult, message):
+    def __init__(self, response, hresult, message, realm):
         self._actual_response_type = None
         self._response_body = None
         self._response = response
         self.hresult = hresult
         self.message = message
-        self._process()
+        self._process(realm.qb_type)
 
-    def _process(self):
+    def _process(self, qb_type):
+        xml_type = utils.get_xml_type(qb_type)
```

Comment:
I actually understand this situation, because after adding POS integration, we will not have static xml type any more. So, maybe refactoring of Processors also needed. Hmm, interesting. The more project gets complex, the more I contemplate that I have not enough time to contribute and enough skills to envision the future path of the project.